WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/LoopingTextWithBackground.kt

```diff
@@ -23,13 +22,15 @@ fun LoopingTextWithBackground(
     modifier: Modifier = Modifier,
     textModifier: Modifier = Modifier,
 ) {
-    Box(modifier.background(colorResource(R.color.bg_jetpack_login_splash))) {
-        Image(
-                painter = painterResource(R.drawable.bg_jetpack_login_splash),
-                contentDescription = stringResource(R.string.login_prologue_revamped_content_description_bg),
-                contentScale = ContentScale.FillBounds,
-                modifier = Modifier.matchParentSize(),
-        )
+    Box(
+            modifier
+                    .background(colorResource(R.color.bg_jetpack_login_splash))
+                    .paint(
+                            painter = painterResource(R.drawable.bg_jetpack_login_splash),
+                            sizeToIntrinsics = true,
+                            contentScale = ContentScale.FillBounds,
+                    )
+    ) {
         LoopingText(
                 modifier = Modifier
                         .fillMaxSize()
```

Comment:
Maybe we could add this here:

```suggestion
                        .clearAndSetSemantics {}
                        .fillMaxSize()
```

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/17317/commits/36c6e4ac85609e65d16e0a2c51773e3c48c73ad9

I was thinking to move `RepeatingColumn` to an upper level package (since initially this was the reasoning I applied to leave the modifier there), but then again it's easy enough to do it when we'd want to reuse it 😄